google-cloud-pubsub/lib/google/cloud/pubsub/service.rb

```diff
@@ -63,7 +63,7 @@ def subscriber
           @subscriber ||= begin
             V1::SubscriberClient.new(
               service_path: host,
-              channel: channel,
+              credentials: channel,
```

Comment:
Same question, is this correct? Seems weird if so.

Reply:
This is also correct. Since Logging needs to set channel args, we can't simply pass the Credentials object. We need to pass a Channels object. We also need to use the `credentials` argument name otherwise the GAPIC code will issue a warning.

Reply:
Also, Pub/Sub supports an emulator, and the way to specify that is using a Channel object for the `credentials` argument. So we have two reasons to do this.